sirepo/template/srw.py

```diff
@@ -1286,6 +1255,52 @@ def _enum_text(name, model, field):
     return ''
 
 
+def _export_rsopt_config(data):
+    v = _rsopt_jinja_context(data.models.exportRsOpt)
+    filename = 'rsOptExport.zip'
+    f = re.sub(r'[^\w.]+', '-', pkio.py_path(filename).purebasename).strip('-')
+    fz = pkio.py_path(filename)
+    v.runDir = f'{f}_scan'
+    v.fileBase = f
+    tf = {k: PKDict(filename=f'{f}.{k}') for k in ['py', 'sh', 'yml']}
+    for t in tf:
+        v[f'{t}FileName'] = tf[t].filename
+    v.outFileName = f'{f}.out'
+    v.readmeFileName = 'README.txt'
+    lf = _SIM_DATA.lib_file_basenames(data)
+    v.libFiles = lf
+    v.hasLibFiles = len(v.libFiles) > 0
+    v.randomSeed = data.models.exportRsOpt.randomSeed if \
+        data.models.exportRsOpt.randomSeed is not None else ''
+
+    # do this in a second loop so v is fully updated
+    # note that the rsopt context is regenerated in python_source_for_model()
+    for t in tf:
+        tf[t].content = python_source_for_model(data, 'rsoptExport', plot_reports=False) \
```

Comment:
The temporary here seems unnecessary. Why not make a function that is called that gets the content below.

Reply:
[075b377](https://github.com/radiasoft/sirepo/pull/4057/commits/075b377ec4e22e26dac2d429ee1d02791ddfeaf1)